### PR TITLE
setting up testing framework 

### PR DESCRIPTION
**Pull request by tochman**
[CLIENT: Set up testing framework](https://www.pivotaltracker.com/story/show/176623679)
### User Story
```
As a development team
In order to be able to make sure our code is doing what it is supposed to do
We would like to be able to write and run automated tests
```
## Changes proposed in this pull request:
* setting baseUrl to "http://localhost:19006/"
* adding viewport size to IPhone X
* adding cypress script ("cypress") and jest script ("test")
* setting chromeWebSecurity to false
* adding jest test to test if app-title exist
## What I have learned working on this feature:
* you can add cypress to native apps
## Packages/Gems added
* cypress
* jest

### CODE REVIEW

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (KyunginNa) [inline cypress/fixtures/example.json]: Looks like this fixture file is not being used. Do you need it?

File: package.json [hunk 2]
```diff
@@ -5,7 +5,9 @@
     "android": "expo start --android",
     "ios": "expo start --ios",
     "web": "expo start --web",
-    "eject": "expo eject"
+    "eject": "expo eject",
+    "cypress": "cypress open",
+    "test": "jest --rootDir=specs --passWithNoTests"
```
Comment (KyunginNa) [inline package.json]: Since you have a test, --passWithNoTests can be deleted?

File: specs/App.spec.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react-native';
+
+import App from '../App';
+
+describe('<App />', () => {
+  it('has 1 child', () => {
+    const screen = render(<App />)
+    expect(screen.getByText('TuneShare').children).toHaveLength(1)
```
Comment (KyunginNa) [inline specs/App.spec.js:9]: What is this testing?🤔 Whether <Text>TuneShare</Text> has children or not???

**Review by KyunginNa — COMMENTED** [review]
Left some comments(mostly questions)!

File: specs/App.spec.js (hunk 3 above)
Comment (ViktorHek) [reply specs/App.spec.js:9]: it's just a test to see if jest is working. after we create our first real test we will delete this one.

Comment (sealfury) [reply specs/App.spec.js:9]: Yeah- until everyone's personal repos are up and running, it's a good barometer to see that whether the dependencies have installed correctly

Comment (KyunginNa) [reply specs/App.spec.js:9]: Oh that's a good idea! I'm convinced👍

**Pull request merged**